packages/lodestar-cli/test/unit/setup.ts

```diff
@@ -0,0 +1,11 @@
+import {rootDir} from "../constants";
```

Comment:
Not a fan of a global setup file that can affect other tests. If you need this steps in the ENR test, put this before and after hooks in the ENR test, and don't use any setup file for the unit tests